### PR TITLE
Final week-1

### PR DESCRIPTION
**Pull request by idanasayag0**

### CODE REVIEW

File: First-Html-page/bonus/bonus.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+}
+body{
+    overflow: hidden;
+    position: relative;
+    text-align: center;
+    justify-content: center;
+    display: flex;
+    background-color: hsl(185, 75%, 39%);
+}
+.background-bottom{
```
Comment (Tamir198) [inline First-Html-page/bonus/bonus.css:14]: Do you think you can find a better name for this? there are a lot of backgrounds and everyone got their own bottom/top.

File: First-Html-page/bonus/index.html [hunk 2]
```diff
@@ -7,29 +7,43 @@
...
-  1.4K
-  Photos
-  <div class="attribution">
-    Challenge by <a href="https://www.frontendmentor.io?ref=challenge" target="_blank">Frontend Mentor</a>. 
-    Coded by <a href="#">Your Name Here</a>.
+  <div class="card">
+    <div class="card-img">
+      <img class="victor-img" src="./images/image-victor.jpg" alt="">
+    </div>
+    <div class="container">
```
Comment (Tamir198) [inline First-Html-page/bonus/index.html:28]: Nice name for this divs class, I understand what it is and what it does.

File: First-Html-page/personal-site-challenge.html [hunk 3]
```diff
@@ -4,26 +4,64 @@
...
-        If you love coding as much as I do.
-        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Adipisci consectetur quisquam deleniti, molestiae reiciendis atque hic blanditiis suscipit voluptates harum ducimus, rerum alias! Quos cum, sequi molestiae itaque id sunt!
-        LinkedIn
-        Twitter
-        Website
-        © DevClub
+        <div class="header">
+            <h1>I'm a Dev Club Member.</h1>
+            <h2>a <span class="under-line">pro</span>grammer</h2>
+            <img  src="./asset/mountain.png" alt="">
```
Comment (Tamir198) [inline First-Html-page/personal-site-challenge.html:16]: You can write some description in the alt attribute

File: First-Html-page/style.css [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom:5rem;
+    border: 0;
+    border-radius: 12px;
+    font-size: 16px;
+    background-color: #11cdd4;
+    color: white;
+    cursor: url(https://cur.cursors-4u.net/games/gam-14/gam1340.cur), auto !important;
```
Comment (Tamir198) [inline First-Html-page/style.css:50]: Why do you have !important here?

**Review by Tamir198 — COMMENTED** [review]
Some general notes:

Try not to use inline styling, style your elements with CSS.
You are mixing a lot of different dimensions units, try to stick to the same unit and use it where you need to.

If you are having some value in your CSS file for example `75px` and you are using it on different elements as lets say `margin-top` you can extract this value into CSS variables

File: First-Html-page/bonus/bonus.css (hunk 1 above)
Comment (idanasayag0) [reply First-Html-page/bonus/bonus.css:14]: Yes, I understand what you say.
If we have big project we need to be more specific.
I'll do this. Thank you

**Pull request closed without merging**